synapse/http/matrixfederationclient.py

```diff
@@ -553,20 +553,20 @@ async def _send_request(
                         with Measure(self.clock, "outbound_request"):
                             # we don't want all the fancy cookie and redirect handling
                             # that treq.request gives: just use the raw Agent.
-                            request_deferred = self.agent.request(
+                            request_deferred = run_in_background(
```

Comment:
It may be worth adding a comment here and below as to why `run_in_background` and `make_deferred_yieldable` are necessary.